### ISSUE TITLE
Changing DOCTYPE

### ISSUE THREAD

**Issue by Lenetta** [react-static/react-static#1586]
Hi, help me, please. 
I`m want using react-static for create emails, how change doctype for html template or using yourself custom index.html?

I`m need using that construction:

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="ru" xmlns="http://www.w3.org/1999/xhtml" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">

**Comment by TriDotS** [comment]
Im not really sure but did you tried [Document](https://github.com/react-static/react-static/blob/master/docs/config.md#document) in `static.config.js`?

**Comment by SleeplessByte** [comment]
That's the way to do it! Implement a custom `Html` inside the `Document` key.

**Comment by Lenetta** [comment]
Yes, I'm tried change Document in static.config.js, it is not changing Doctype. It is changing only html tag and nested elements

**Comment by SleeplessByte** [comment]
Oh that's right. We do that manually, see:

https://github.com/react-static/react-static/blob/8661f939be5cb15469c7cdaedcfd640a9bc1724a/packages/react-static/src/static/exportRoute.js#L261-L263

https://github.com/react-static/react-static/blob/3793552f6e9ff921cb8480768a7438e386f6194b/packages/react-static/src/utils/createIndexPlaceholder.js#L22-L25

However, that first one shows something you might be able to use: `beforeDocumentToFile` hook is passed in the raw `html`. Hoping the placeholder index isn't actually used when you're exporting the emails.
